libs/designsystem/src/lib/components/dropdown/dropdown.component.ts

```diff
@@ -233,6 +233,19 @@ export class DropdownComponent implements AfterViewInit, OnDestroy, ControlValue
 
   onToggle(event: MouseEvent) {
     event.stopPropagation();
+
+    /* The 'clicked' class is applied to prevent the dropdown from getting a focus ring on click.
+    There is a bug that causes the dropdown to get a focus ring on click, if it is the first element that is interacted with
+    after the page is loaded. If the user interacts with any other element before, then the dropdown won't get a focus ring.
+    See issue: https://github.com/kirbydesign/designsystem/issues/2477.
+
+    This solution can potentially be refactored, when popover is not experimental anymore. Then it could be possible 
+    to close the dropdown when the popover backdrop is clicked, instead of relying on the blur event, which is utilized
+    by this line below: this.elementRef.nativeElement.focus(). Right now this forces the blur event to be triggered, when
+    clicking outside of the dropdown.
+    */
+    this.elementRef.nativeElement.classList.add('clicked');
```

Comment:
could this logic be done with a hostbinding and a boolean instead?

Reply:
Good idea! I have made the changes :)